README.rst

```diff
@@ -32,57 +32,40 @@ The pre-configured template for Thamos CLI is available in the
 
   cat .thoth.yaml
 
-Alternatively, to generate Thoth's configuration file out of the template run the
-following command:
-
-.. code-block:: console
-
-  thamos config --no-interactive --template thoth_conf_template.yaml
-  cat .thoth.yaml  # to browse the content of the config file
-
 Now you are ready to ask for advises:
 
 .. code-block:: console
 
   thamos advise
 
 This might take some time. Once Thoth recommends you the application stack to
-be used for running the application, create a Python environment and install
-requirements into it:
+be used for running the application, you can use Thamos to create a Python
+environment (based on configuration in ``.thoth.yaml``) and install the
+recommended requirements into it:
 
 .. code-block:: console
 
-  pip3 install micropipenv
+  thamos install
```

Comment:
Is `micropipenv` already in the requirements?

Reply:
Yes, it is a dependency of thamos:

https://github.com/thoth-station/thamos/blob/2fe398dcfbeb8477305aa285be550e565e9a51a8/requirements.txt#L18

But the user does not need to use it at all when using thamos (thamos wraps it with a few checks to properly integrate it).